lib/ansible/modules/windows/win_description.ps1

```diff
@@ -0,0 +1,56 @@
+#!powershell
+
+# Copyright: (c) 2019, RusoSova
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#AnsibleRequires -CSharpUtil Ansible.Basic
+#AnsibleRequires -OSVersion 6.0
```

Comment:
This is the default, no need to have this line unless you declare a version that is newer than this.